src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/OverridePimStyles.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const OverridePimStyle = () => {
+    const style = {
+        __html: `
+        .AknDefault-mainContent {
+            padding-bottom: 0px;
+        }`
+    };
+
+    return (
+        <style dangerouslySetInnerHTML={style} />
+    );
```

Comment:
You can simplify like this:

```suggestion
    const style = `
        .AknDefault-mainContent {
            padding-bottom: 0px;
        }
    `;

    return (
        <style>{style}</style>
    );
```